Makefile

```diff
@@ -36,6 +38,14 @@ stressmeta: override STRESSFLAGS += -p 1
 stressmeta: override TESTS = TestMeta$$
 stressmeta: stress
 
+.PHONY: crossversion-meta
+crossversion-meta:
+	git checkout ${LATEST_RELEASE}; \
+		${GO} test -c ./internal/metamorphic -o './internal/metamorphic/crossversion/${LATEST_RELEASE}.test'; \
```

Comment:
Might be worth adding these `./internal/metamorphic/*.test` binaries to the `.gitignore`, and / or doing some cleanup at the end of a run.